### PR TITLE
commit for sdevopsadm/roadmap#26 and Unit Test for vet service has be…

### PR DESCRIPTION
**Pull request by Mike-msoh**
…en revised (sdevopsadm/roadmap#14)

Fix#1
Error Code has been inserted in Java source for Vet Service and Visit Service. 
Finebug code has been tested with findbugs as devops command.

Fix#2
Unit Test for Vet Service is not merged. Please merge VetResourceVerifyTest.java

### CODE REVIEW

File: spring-petclinic-vets-service/src/test/java/org/springframework/samples/petclinic/vets/web/VetResourceVerifyTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+/**
+ ** @author msoh
+ */
+@RunWith(SpringRunner.class)
+@DataJpaTest
+@ActiveProfiles("test")
+public class VetResourceVerifyTest {
+
+	@Autowired
```
Comment (jgkong) [inline spring-petclinic-vets-service/src/test/java/org/springframework/samples/petclinic/vets/web/VetResourceVerifyTest.java]: This test change is conflict with previous PR.

**Comment by Mike-msoh** [comment]
The conflict has been resolved.

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/673651ec-45f6-4e57-977b-ac018a92f419 "PoC2-vets-service-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170524-181327 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170524-181327") to [POC](https://console.ng.bluemix.net/apps/e0ff7bdd-8955-4b8a-81ba-cc3bcda0f41d?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:POC"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/673651ec-45f6-4e57-977b-ac018a92f419 "PoC2-vets-service-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-020956 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-020956") to [POC]( "ibm:yp:us-south:VDInfraPOC:POC"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/5e65a470-ac27-4ff4-9123-1ce97f9f44db "PoC2-api-gateway-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-024129 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-024129") to [POC](https://console.ng.bluemix.net/apps/0e2c13c4-54ec-45ed-9274-834bec850d52?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:POC"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/b6b043b3-d9a4-4d27-b03c-cf5fa42abdc1 "PoC2-customers-service-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-033717 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-033717") to [POC](https://console.ng.bluemix.net/apps/6caea5ce-af1d-4d76-a573-7a0c9156e4bf?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:POC"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/5e65a470-ac27-4ff4-9123-1ce97f9f44db "PoC2-api-gateway-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-034706 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-034706") to [POC]( "ibm:yp:us-south:VDInfraPOC:POC"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/b6b043b3-d9a4-4d27-b03c-cf5fa42abdc1 "PoC2-customers-service-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-035408 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-035408") to [POC](https://console.ng.bluemix.net/apps/6caea5ce-af1d-4d76-a573-7a0c9156e4bf?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:POC"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/b6b043b3-d9a4-4d27-b03c-cf5fa42abdc1 "PoC2-customers-service-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-041018 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-041018") to [POC]( "ibm:yp:us-south:VDInfraPOC:POC"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/b6b043b3-d9a4-4d27-b03c-cf5fa42abdc1 "PoC2-customers-service-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170607-084900 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170607-084900") to [POC](https://console.ng.bluemix.net/apps/6caea5ce-af1d-4d76-a573-7a0c9156e4bf?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:POC"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/5e65a470-ac27-4ff4-9123-1ce97f9f44db "PoC2-api-gateway-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170607-101114 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170607-101114") to [POC]( "ibm:yp:us-south:VDInfraPOC:POC"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/673651ec-45f6-4e57-977b-ac018a92f419 "PoC2-vets-service-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170609-062716 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170609-062716") to [POC](https://console.ng.bluemix.net/apps/e0ff7bdd-8955-4b8a-81ba-cc3bcda0f41d?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:POC"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/5e65a470-ac27-4ff4-9123-1ce97f9f44db "PoC2-api-gateway-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-dev-20170629-041513 "deploy-ibm-yp-us-south-VDInfraPOC-dev-20170629-041513") to [dev](https://console.bluemix.net/apps/a4a4c98c-f00c-41f5-a60c-4751e5da00e0?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:dev"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/673651ec-45f6-4e57-977b-ac018a92f419 "PoC2-vets-service-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-dev-20170629-042646 "deploy-ibm-yp-us-south-VDInfraPOC-dev-20170629-042646") to [dev](https://console.bluemix.net/apps/076deb48-e8d0-4c56-b906-7cca0e7090da?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:dev"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/5e65a470-ac27-4ff4-9123-1ce97f9f44db "PoC2-api-gateway-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-qa-20170629-043922 "deploy-ibm-yp-us-south-VDInfraPOC-qa-20170629-043922") to [qa]( "ibm:yp:us-south:VDInfraPOC:qa"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/673651ec-45f6-4e57-977b-ac018a92f419 "PoC2-vets-service-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-qa-20170629-051215 "deploy-ibm-yp-us-south-VDInfraPOC-qa-20170629-051215") to [qa]( "ibm:yp:us-south:VDInfraPOC:qa"), including commit 7dd98e8c2a44cb78c9b1c8fea11f8df03f92f055

**Pull request merged**